core/src/main/java/org/elasticsearch/node/service/NodeService.java

```diff
@@ -135,7 +135,8 @@ public NodeInfo info() {
                 transportService.info(),
                 httpServer == null ? null : httpServer.info(),
                 pluginService == null ? null : pluginService.info(),
-                ingestService == null ? null : ingestService.info()
+                ingestService == null ? null : ingestService.info(),
+                indicesService == null ? null : indicesService.getTotalIndexingBufferBytes()
```

Comment:
the node stats method is happily using indicesService as is
                indicesService.stats(true)

I think it can never be null.